docs/index.md

```diff
@@ -1,7 +1,28 @@
 # Faithlife.Data
 
+**Faithlife.Data** provides helpers for querying ADO.NET-compatible databases.
+
 [![NuGet](https://img.shields.io/nuget/v/Faithlife.Data.svg)](https://www.nuget.org/packages/Faithlife.Data)
 
-## Usage
+## Overview
+
+The **Faithlife.Data** class library provides an enhanced API for interacting with ADO.NET-compatible databases. It is similar to [Dapper](https://github.com/StackExchange/Dapper) and other "micro" ORMs for .NET.
+
+To use this library, call [`DbConnector.Create()`](Faithlife.Data/DbConnector/Create.md) to create a [`DbConnector`](Faithlife.Data/DbConnector.md) with a valid `IDbConnection` from your favorite ADO.NET database provider, e.g. [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient/) for SQL Server or [MySqlConnector](https://www.nuget.org/packages/MySqlConnector/) for MySQL.
```

Comment:
`Microsoft.Data.SqlClient` is the new client package for SQL Server.

Reply:
:+1: for MySqlConnector.

Reply:
See announcement blog post: https://devblogs.microsoft.com/dotnet/introducing-the-new-microsoftdatasqlclient/

> The Microsoft.Data.SqlClient package ... will be the flagship data access driver for SQL Server going forward.

Reply:
Fixed, thanks!